packages/templates/src/raw/album-releases/template.json

```diff
@@ -3523,7 +3523,7 @@
           "lineHeight": 1,
           "textAlign": "initial",
           "padding": { "locked": true, "hasHiddenPadding": false },
-          "content": "<span style=\"color: #fee2c6; letter-spacing: 0.1em\">THAT's</span>\n<span style=\"color: #fee2c6; letter-spacing: 0.1em\">FLOW!</span>",
+          "content": "<span style=\"color: #fee2c6; letter-spacing: 0.1em; text-transform: uppercase\">That<span style=\"color: #fee2c6; letter-spacing: 0.1em\">'s</span></span>\n<span style=\"color: #fee2c6; letter-spacing: 0.1em; text-transform: uppercase\">Flow!</span>",
```

Comment:
`'s` too need to be a part of the TT as per design

Reply:
not sure that, `s` looks smaller 🤔